_posts/2023-02-15-#26441.md

```diff
@@ -10,15 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- A Bitcoin peer may have connections with other peers. By default (in Bitcoin Core), it allows up to 125 connections, 
+  8 of which are outbound.
+
+- There are different kind of connections. E.g: block-relay-only, inbound, outbound-full-relay, manual, feeler, 
+  block-relay and addr-fetch.
+
+- You can use RPC `getpeerinfo` to get data about each connected network peer and check the connection type, permissions
+  and other informations.
+
+- `-whitelist` flag allows to add permission flags to the peers connecting from the given IP address or CIDR-notated
+  network. It uses the same permissions as `-whitebind`: bloomfilter, noban, forcerelay, relay, mempool, download,
+  addr).
+
+- #17167 proposed to change `-whitelist` to allow whitelisting outgoing connections.
+
 ## Questions
-1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
--->
 
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? 
+   What was your review approach?
+
+1. What does this PR do?
+
+1. `-whitelist` only allows to add permission flags to inbound peers. Why only for inbound ones?
+   
+1. Does it make sense to extend the permissions to outbound peers? Why?
```

Comment:
Isn't this the same as the previous question?

Reply:
Not the same, but I think we can merge them.